File: telemetry/models/streamable/StreamablePlayerCarInfo.py
from dataclasses import dataclass

"""
Info about the car of the user.
"""


# Streaming data (websocket)
@dataclass
class StreamablePlayerCarInfo:
    brakeInput: float
    absActivated: bool
    throttleInput: float
    rpm: int
    speed: float
    gear: int
    fuelLevel: float
    fuelPercentage: float

File: server/src/main/kotlin/com/landonpatmore/models/streamable/receivable/StreamablePlayerCarInfo.kt
package com.landonpatmore.models.streamable.receivable

data class StreamablePlayerCarInfo(
    val brakeInput: Float,
    val absActivated: Boolean,
    val throttleInput: Float,
    val rpm: Int,
    val speed: Float,
    val gear: Int,
    val fuelLevel: Float,
    val fuelPercentage: Float
)
